Finding the Numbers

narayan2111 opened the pull request:
<!--
Thank you for sending the PR! We appreciate you spending the time to work on these changes.

Help us understand your motivation by explaining why you decided to make this change.
You can learn more about contributing to NeoAlgo here: https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md

Happy Contributing!

-->

### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?


Yes

### Description

Program to find the Unique numbers in an array out of which two numbers occur exactly once and are distinct.
You need to find the other two numbers and print them in ascending order.

Also Taken help from geeksforgeeks


### Checklist

- [Y] I've read the contribution guidelines.
- [Y] I've checked the issue list before deciding what to submit.
- [Y] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#922

narayan2111 committed:
Added Find the numbers

narayan2111 committed:
Update README.md

narayan2111 commented:
kindly review this PR @iamrajiv @plazzy99 @Kajol-Kumari

vatsalkesarwani12 reviewed (changes requested):
https://www.geeksforgeeks.org/find-unique-pair-array-pairs-numbers/

code is exactly same with comments and all

narayan2111 committed:
Updated the Code

narayan2111 committed:
Added the outer source help description

narayan2111 commented:
> https://www.geeksforgeeks.org/find-unique-pair-array-pairs-numbers/
> 
> code is exactly same with comments and all

@plazzy99 
Updated the whole code and comments but algorithm is same.
Also I added geeksforgeeks in comment from where i had taken help

narayan2111 commented:
Kindly review this PR @Kajol-Kumari @iamrajiv @plazzy99

Kajol-Kumari commented on C/other/Find_the_numbers.c:
```diff
@@ -0,0 +1,61 @@
+//Program to find the Unique numbers in an array out of which two numbers occur exactly once and are distinct.
```
please use multiline comment here.

Kajol-Kumari reviewed (approved):
@narayan2111 Minor comment, Rest LGTM :+1:

narayan2111 commented:
Kindly review this PR @plazzy99
